vega_sim/scenario/ideal_market_maker_v2/agents.py

```diff
@@ -53,7 +53,9 @@ def __init__(
         asset_decimal: int = 5,
         market_decimal: int = 5,
         market_position_decimal: int = 2,
-        commitment_amount: float = 6000,
+        market_name: str = None,
+        asset_name: str = None,
+        commitment_amount: float = 20000000000,
```

Comment:
I think commitment amount 6000 is probably a more standard 'baseline' than the agent committing $20bn to the market (also slightly concerning if that ends up necessary for a 'realistic' simulation?)

Reply:
Yeah, here is just a default value, will change it to a normal level.